src/Http/Identifiers.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Kirby\Http;
+
+use Kirby\Toolkit\Obj;
+
+/**
+ * A wrapper for URL query and params
+ * @internal
+ *
+ * @package   Kirby Http
+ * @author    Nico Hoffmann <nico@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://opensource.org/licenses/MIT
+ */
+abstract class Identifiers extends Obj
```

Comment:
@bastianallgeier @lukasbestle if you have any idea for a better name, please let me know

Reply:
Maybe `KeyValueData`?

Reply:
Mmm that sounds super general. Maybe it's not even worth it to have a common parent class for keeping it DRY just a little bit more. Maybe you creating both classes separately with these few methods existing in both would be the cleaner way after all.

Reply:
I am not sure. I'm just thinking out loud. Don't you think it is more appropriate to use `Trait` instead `abstract class`  in this context? 🤔

Reply:
@afbora what would be the benefit?

Reply:
Great question 😄 To be honest, I can't show any obvious benefit. It just made more sense to me to use trait in terms of its function.

Does having several methods be similar require inheritance? Even the constructor of `Query` and `Params` is different anyway. `Identifiers::__construct` didn't seem very functional. As a result `Identifiers` class seemed like an unnecessary layer of inheritance to me. Not important. It's just that this extra layer was a bit more tiring when reviewing the code.

Reply:
Yea but that's be the same as a trait. Really leaning much more towards putting the same code just in the two classes. Tiny bit less DRY but so much better to read I think.

Reply:
I agree.

Reply:
✅